runusb

```diff
@@ -204,117 +205,65 @@ class RobotUSBHandler(USBHandler):
-            "--ports", "2",  # only switch ports 2-5
-            "--action", "cycle",
-            "--delay", "1",  # time to wait before powering back on
-        ]
-        LOGGER.info("power-cycling USB bus to reset servo board (srobo/servo-v4-fw#7)")
-        try:
-            subprocess.run(command, check=True)
-        except BaseException:
-            LOGGER.exception("failed to power-cycle USB bus")
-
 
 class MetadataUSBHandler(USBHandler):
-    def __init__(self, mountpoint_path: str):
+    def __init__(self, mountpoint_path: str) -> None:
         pass  # Nothing to do.
 
     def close(self) -> None:
         pass  # Nothing to do.
 
 
 class AutorunProcessRegistry(object):
-    TYPE_HANDLERS = {
+    TYPE_HANDLERS: dict[USBType, Type[USBHandler]] = {
         USBType.ROBOT: RobotUSBHandler,
         USBType.METADATA: MetadataUSBHandler,
-    }  # type: Dict[USBType, Type[USBHandler]]
+    }
 
     def __init__(self) -> None:
-        self.mountpoint_handlers = {}  # type: Dict[str, USBHandler]
-        self.mountpoint_types = {}  # type: Dict[str, USBType]
+        self.mountpoint_handlers: dict[str, USBHandler] = {}
 
     def update_filesystems(self, mountpoints: Iterator[Mountpoint]) -> None:
-        actual_mountpoint_paths = {
+        actual_mountpoints = {
             x.mountpoint
             for x in mountpoints
             if self._is_viable_mountpoint(x)
         }
 
-        expected_mountpoint_paths = set(self.mountpoint_handlers.keys())
+        expected_mountpoints = set(self.mountpoint_handlers.keys())
 
         # Handle newly detected filesystems
-        for new_mountpoint_path in (
-            actual_mountpoint_paths
-            - expected_mountpoint_paths
-        ):
-            self._detect_new_mountpoint_path(new_mountpoint_path)
+        for new_mountpoint in (actual_mountpoints - expected_mountpoints):
+            self._detect_new_mountpoint_path(new_mountpoint)
 
         # Handle now-dead filesystems
-        for old_mountpoint_path in (
-            expected_mountpoint_paths
-            - actual_mountpoint_paths
-        ):
-            self._detect_dead_mountpoint_path(old_mountpoint_path)
+        for old_mountpoint in (expected_mountpoints - actual_mountpoints):
+            self._detect_dead_mountpoint_path(old_mountpoint)
 
     def _detect_new_mountpoint_path(self, path: str) -> None:
-        LOGGER.info("Found new mountpoint: %s", path)
         usb_type = detect_usb_type(path)
+        LOGGER.info(f"Found new mountpoint: {path} ({usb_type})")
         handler_class = self.TYPE_HANDLERS[usb_type]
         handler = handler_class(path)
         LOGGER.info("  -> launched handler")
         self.mountpoint_handlers[path] = handler
-        self.mountpoint_types[path] = usb_type
 
     def _detect_dead_mountpoint_path(self, path: str) -> None:
-        LOGGER.info("Lost mountpoint: %s", path)
+        LOGGER.info(f"Lost mountpoint: {path}")
```

Comment:
Nit: The former is better and preferred, as Python doesn't need to interpolate the string if the log message isn't output.

Reply:
True but f-strings make the lines more readable, the overhead is minimal and these lines aren't logged frequently enough for it to be an issue.